build.psm1

```diff
@@ -489,9 +489,13 @@ Fix steps:
     }
 
     # handle TypeGen
-    if ($TypeGen -or -not (Test-Path "$PSScriptRoot/src/System.Management.Automation/CoreCLR/CorePsTypeCatalog.cs")) {
+    $incFileName = 'powershell'
+    if (-not [string]::IsNullOrEmpty($Runtime)) {
+        $incFileName = "$($incFileName)_$($Runtime.Substring(0,3))" # use the first 3 characters only because otherwise dotnet will crash due to the separation characters
```

Comment:
I believe our guidance is to put the comment on a separate line.

Reply:
you could replace the seperation character.... not sure it's needed though.

Reply:
You're right about the comment. I noticed that only after the commit but I will fix that. The separation character is only for readability because to me `powershell_win` or `powerhell_lin` is easier and faster to read than `powershellwin` or `powershelllin`